README.md

```diff
@@ -44,7 +44,7 @@ For local development of Terraform core, first make sure Go is properly installe
 Next, using [Git](https://git-scm.com/), clone this repository into `$GOPATH/src/github.com/hashicorp/terraform`.
 
 You'll need to run `make tools` to install some required tools, then `make`.  This will compile the code and then run the tests. If this exits with exit status 0, then everything is working!
-You only need torun `make tools` once (or when the tools change).
+You only need to run `make tools` once (or when the tools change).
```

Comment:
I know this kind of violates the spirit of single purpose PRs, but I didn't want to open another one just to add this space.